### DISCUSSION TITLE
dont accept ex twitter engineer code contributions

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#19639] [General]
elon said he will not allow former engineers to add twitter code to mastodon

**Comment by Gargron** [comment]
Source?

**Reply by ghost** [reply]
that ex twitter engineer who complained to elon said he was going to retaliate by contributing to mastodon

**Reply by Gargron** [reply]
I mean where has Elon said that?

**Reply by ghost** [reply]
https://www.wired.com/story/twitter-open-algorithm-problem/

**Reply by Gargron** [reply]
Ctrl+F "mastodon" yields 0 results on that article. In any case, I have no reason to prevent contributions from anyone just because Elon Musk said so, if there are legal obstacles on the ex-employees' side that's between them and Elon.

**Comment by ghost** [comment]
https://www.wired.com/story/twitter-open-algorithm-problem/
